skimage/measure/tests/test_regionprops.py

```diff
@@ -704,6 +704,13 @@ def test_intensity_mean():
     assert_almost_equal(intensity, 1.02777777777777)
 
 
+def test_intensity_median():
+    intensity = regionprops(SAMPLE, intensity_image=INTENSITY_SAMPLE)[
+        0
+    ].intensity_median
+    assert_almost_equal(intensity, 1.0)
```

Comment:
Perhaps test on a more meaningful input image since `INTENSITY_SAMPLE` only contains 0 and 1?